### PR TITLE
verify if $the_['function'] is not an empty string ""

### PR DESCRIPTION
**Pull request by herizo**
Fix for the error on line 214  because $the_['function'] was an empty string ""

### CODE REVIEW

File: wp-includes/plugin.php [hunk 1]
```diff
@@ -208,7 +208,7 @@ function apply_filters( $tag, $value ) {
 
 	do {
 		foreach( (array) current($wp_filter[$tag]) as $the_ )
-			if ( !is_null($the_['function']) ){
+			if ( !is_null($the_['function']) && $the_['function'] != '' ){
```
Comment (septyaman) [inline wp-includes/plugin.php]: Why not use 
``php
!empty($the_['function'])
````

Comment (herizo) [inline wp-includes/plugin.php]: i don't remember  the exact use case  but now you ask the question '!empty()' seems a better solution.

**Comment by zamber** [comment]
The description of this repo is: WordPress, Git-ified. Synced via SVN every 15 minutes, including branches and tags! This repository is just a mirror of the WordPress subversion repository. Please do not send pull requests. Submit patches to http://core.trac.wordpress.org/ instead.

**Comment by septyaman** [comment]
:+1:

**Pull request closed without merging**